src/ServiceControl.Monitoring/MetricReport.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.Metrics
+{
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// The reporting message.
+    /// </summary>
+    public class MetricReport : ICommand
```

Comment:
I think we should de-serialize directly to JsonMetricsContext instead of JObject. It should be possible if we use Newtonsoft serializer as daniel pointed out.